apps/web/src/pages/block/[id].tsx

```diff
@@ -85,7 +86,11 @@ const Block: NextPage = function () {
 
     detailsFields = [
       {
-        name: "Block Height",
+        name: (
+          <InfoField description="The block height is the number of blocks preceding the current block.">
```

Comment:
Maybe it’s just me, but I find this a little bit confusing. Based on this description, does having block number 10 mean that 10 blocks precede it?

I suggest the following alternative description:

```suggestion
          <InfoField description="Also referred to as Block Number, the block height represents the length of the blockchain and increases with each newly added block.">
```